### PR TITLE
RENAME geo:proximity -> geo:distance

### PR DESCRIPTION
**Pull request by fgalan**
@kzangeli 
@crbrox 
@jmcanterafonseca (nice to have)

### CODE REVIEW

**Comment by kzangeli** [comment]
LGTM

File: doc/apiary/v2/fiware-ngsiv2-reference.apib [hunk 1]
```diff
@@ -344,8 +344,8 @@ The following strings must not be used as attribute names:
 Operations that retrieve lists of entities allow the `orderBy` URI parameter to specify the order; its 
 value can be:
 
-* The keyword "geo:proximity" to order results by distance to the center point when combined with "near" geoqueries.
-  If "geo:proximity" is used without "near" geoqueries, the order is undefined.
+* The keyword "geo:distance" to order results by distance to the center point when combined with "near" geoqueries.
```
Comment (jmcanterafonseca) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: better 

The keyword ```geo:distance```  to order results by distance to a reference geometry when a "near" (```georel=near```) spatial relationship is used.

**Comment by jmcanterafonseca** [comment]
LGTM with the tiny comment addressed

thanks!

File: doc/apiary/v2/fiware-ngsiv2-reference.apib (hunk 1 above)
Comment (fgalan) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: Fixed in f1c3ec825e826b17b58ddd7a258daabfbdda31a2

**Comment by crbrox** [comment]
LGTM

**Pull request merged**